plugins/api-docs/src/components/useComponentApiEntities.ts

```diff
@@ -43,9 +47,12 @@ export function useComponentApiEntities({
     await Promise.all(
       apiNames.map(async name => {
         try {
+          const apiEntityRef = parseEntityRef(name);
```

Comment:
@K-Phoen  Hi! This is close but needs a little tweak.

The pattern to use is to do `parseEntityName(name, { defaultNamespace: entity.metadata.namespace, defaultKind: 'API' })`. What you get out then, is a fully qualified name always. Then you will have to check that the thing that came out actually had `kind` equal to `'API'` and reject it if it isn't (the user entered a reference to the wrong kind of entity unexpectedly).

That name, can then be given directly to `catalogApi.getEntityByName`.

Reply:
So close :D

I updated the PR, thanks for your feedback!